src/ardrive.ts

```diff
@@ -499,9 +526,16 @@ export class ArDrive extends ArDriveAnonymous {
 			throw new Error(errorMessage.entityNameExists);
 		}
 
+		const conflictingFileName = filesAndFolderNames.files.find((f) => f.fileName === destFileName);
+
+		if (conflictingFileName && conflictResolution === 'skip') {
+			return emptyArFSResult;
+		}
+		// TODO: Handle this.conflictResolution === 'upsert' and 'ask' PE-638
```

Comment:
@arielmelendez @agsuy @fedellen 
Do you guys think we need another story ticket for it? How are we gonna deal with these cupcakes at the process level?

Reply:
I know we talked about it at some standup, but I don't recall :sweat_smile: